.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: csharp
+
+matrix:
+  include:
+    # - os: osx
+    #   osx_image: xcode7.3
+    #   dotnet: 1.0.0-preview2-003131
+    #   mono: none
+    - os: linux
+      dist: trusty
+      dotnet: 1.0.0-preview2-003131
+      mono: none
+      sudo: required
+
+notifications:
+  email:
+    on_success: change
+    on_failure: always
+
+script:
+  time dotnet restore && dotnet build src/Anat -f netstandard1.6 && dotnet test tests/Anat.Tests -f netcoreapp1.0
```

Comment:
i think is better to always build in release mode in ci server, because 
- it's not build usually by user (catch compilation error)
- the built package is ready to use
you can add `-c Release` to both `build` and `test`

Also you can just `time dotnet restore && dotnet test tests/Anat.Tests -f netcoreapp1.0 -c Release` because if not up to date, dotnet test will build the project and referenced projects